code/obj/mining.dm

```diff
@@ -398,8 +415,6 @@
 
 		small
 			marker_type = /obj/magnet_target_marker/small
-			get_encounter(rarity_mod)
-				return mining_controls.select_small_encounter(rarity_mod)
```

Comment:
like, since you removed the proc, will this even work properly now? You removed the proc without accounting for different behavior here

Reply:
Get_encounter is never called in any file according to my search.

Reply:
Actually, I had thought the small magnet was entirely non-functional already. I had never seen one in game. I've spawned and tested the small magnet and it is a working thing, even if it isn't naturally attainable. I'll test and commit the necessary fixes in case the small magnet is ever re-introduced.

Reply:
Alright, I've reverted the get_encounter removal and called it at the relevant points. All three types of magnet have been tested and work with random asteroids and telescope asteroids.